src/librustdoc/html/render.rs

```diff
@@ -1556,6 +1557,7 @@ impl AllTypes {
             typedefs: HashSet::with_capacity(100),
             statics: HashSet::with_capacity(100),
             constants: HashSet::with_capacity(100),
+            keywords: HashSet::with_capacity(10),
```

Comment:
Wondering if this should join the rest at 100 default capacity, since there are currently [62 valid elements](https://github.com/rust-lang/rust/blob/2612bbcba08dd81730edd8f2139005fb7a409294/src/libsyntax_pos/symbol.rs#L328-L404) in the `Keyword` enum, which could feasibly each get documentation.

Reply:
Let's put 62 then. :D